src/core/Microsoft.PowerToys.Settings.UI.UnitTests/Mocks/ISettingsUtilsMocks.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.PowerToys.Settings.UI.Lib;
+using Microsoft.PowerToys.Settings.UI.Lib.Utilities;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerToys.Settings.UI.UnitTests.Mocks
+{
+    internal static class ISettingsUtilsMocks
+    {
+        //Stubs out empty values for imageresizersettings and general settings as needed by the imageresizer viewmodel
+        internal static Mock<ISettingsUtils> GetStubSettingsUtils()
+        {
+            var settingsUtils = new Mock<ISettingsUtils>();
+            settingsUtils.Setup(x => x.GetSettings<GeneralSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new GeneralSettings());
+            settingsUtils.Setup(x => x.GetSettings<ImageResizerSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new ImageResizerSettings());
+            settingsUtils.Setup(x => x.GetSettings<ColorPickerSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new ColorPickerSettings());
+            settingsUtils.Setup(x => x.GetSettings<KeyboardManagerSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new KeyboardManagerSettings());
+            settingsUtils.Setup(x => x.GetSettings<PowerLauncherSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new PowerLauncherSettings());
+            settingsUtils.Setup(x => x.GetSettings<PowerPreviewSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new PowerPreviewSettings());
+            settingsUtils.Setup(x => x.GetSettings<PowerRenameLocalProperties>(It.IsAny<string>(), It.IsAny<string>())).Returns(new PowerRenameLocalProperties());
+            settingsUtils.Setup(x => x.GetSettings<GeneralSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new GeneralSettings());
+            settingsUtils.Setup(x => x.GetSettings<FancyZonesSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new FancyZonesSettings());
+            settingsUtils.Setup(x => x.GetSettings<ShortcutGuideSettings>(It.IsAny<string>(), It.IsAny<string>())).Returns(new ShortcutGuideSettings());
```

Comment:
Great callout.  I updated this method to leverage the It.IsAnyType. One interesting thing (I can't explain) is that for some reason the *Settings object that was previously getting returned seemed to be reused somehow.  When I modified it to use `Activator.CreateInstance(typeArgument);` instead of `new *Settings();` it exposed some tests that needed a mocked IO provider instead of a simple stub.   I've updated those tests as well.